src/hotspot/cpu/x86/stubGenerator_x86_64.cpp

```diff
@@ -7000,7 +7001,18 @@ address generate_avx_ghash_processBlocks() {
     // Get svml stub routine addresses
     void *libsvml = NULL;
     char ebuf[1024];
-    libsvml = os::dll_load(JNI_LIB_PREFIX "svml" JNI_LIB_SUFFIX, ebuf, sizeof ebuf);
+    char dll_path[JVM_MAXPATHLEN];
+#ifdef _WINDOWS
+    int ret = jio_snprintf(dll_path, sizeof(dll_path), "%s%sbin", Arguments::get_java_home(), os::file_separator());
+#else
+    int ret = jio_snprintf(dll_path, sizeof(dll_path), "%s%slib", Arguments::get_java_home(), os::file_separator());
+#endif
```

Comment:
There is a platform independent way to do this. Here is the example of how libverify is loaded by the JVM:

// Load verify dll
  char buffer[JVM_MAXPATHLEN];
  char ebuf[1024];
  if (!os::dll_locate_lib(buffer, sizeof(buffer), Arguments::get_dll_dir(), "verify"))
    return NULL; // Caller will throw VerifyError

  void *lib_handle = os::dll_load(buffer, ebuf, sizeof(ebuf));